docs/devguide.md

```diff
@@ -25,8 +25,9 @@ layout:
     │   └── catalog             # Helm chart for deploying the service catalog
     │   └── ups-broker          # Helm chart for deploying the user-provided service broker
     ├── cmd                     # Contains "main" Go packages for each service catalog component binary
-    │   └── apiserver           # The service catalog API server binary
-    │   └── controller-manager  # The service catalog controller manager binary
```

Comment:
If we don't have these directories capable of making binaries, this should probably be moved either to `pkg` or under the `service-catalog` package.